### ISSUE TITLE
Lost Audio Arch latest pipewire/wireplumber/kernel updates. Workaround. (LoL-git script) FYI!!!

### ISSUE THREAD

**Issue by caseyjp11** [kyechou/leagueoflegends#40]
Workaround:
tl;dr: switch wine's default audio output to ALSA. (winetricks or other methods).

So. Updating to ze bleeding edge once in a while leads to issues. heh.

After upgrading my Arch system to the latest updates today I lost wine audio/LoL audio.

Updates suspected to be involved:

kernel: 5.16.12-arch1-1

extra/pipewire 1:0.3.48-1 (1.6 MiB 7.7 MiB) (Installed)

extra/wireplumber 0.4.8-1 (381.6 KiB 2.1 MiB) (Installed)

as well as various GStreamer updates.

-------------

LoL completely lost audio. The winecfg audio tab 'hangs' for 10-15 seconds before switching, but then the test sound button works.

The error spit out in journalctl is:

wireplumber[897]: <WpSiAudioAdapter:0x5636567d47e0> no usable format found for node 77

wireplumber[897]: <WpSiAudioAdapter:0x5636567d47e0> failed to configure item for node 77

This corresponds to the time that LoL finishes connecting to the port and the client launches. The client seems to hang for a bit, then comes up normally, but there is no audio.

The solution for me was to use winetricks to change wine's default sound driver to "ALSA".

Rerunning winecfg showed the audio tab responsive as it should be and test audio worked.

Restarting LoL then proceeded as it should and full audio returned.

I don't have the first clue which component is causing this node issue, but this seems to fix things for now.

**Comment by kyechou** [comment]
Closing for now, assuming the fix is going to be released soon (or have been released already). Ref: https://www.reddit.com/r/leagueoflinux/comments/t7ob4u/comment/hzkaa3h/?utm_source=share&utm_medium=web2x&context=3
